### PR TITLE
Add example_results() fixture to conftest.py file

### PR DESCRIPTION
**Pull request by dorothykiz1**
xref #1030

### CODE REVIEW

File: test/conftest.py [hunk 1]
```diff
@@ -5,7 +5,9 @@
 from asv import config
 from asv import repo
 from .test_workflow import generate_basic_conf
-
+import shutil
+from os.path import abspath, dirname
```
Comment (datapythonista) [inline test/conftest.py:9]: The imports from Python's standard library should come first.

**Review by datapythonista — COMMENTED** [review]
Looks good, just one minor comment, and also you've got conflicts to solve. And let's see if the CI is happy.

File: test/conftest.py (hunk 1 above)
Comment (dorothykiz1) [reply test/conftest.py:9]: Let me fix them

**Comment by dorothykiz1** [comment]
@datapythonista this PR is ready for your review

**Comment by datapythonista** [comment]
Thanks @dorothykiz1, really nice clean up!

**Pull request merged**